src/engraving/dom/undo.cpp

```diff
@@ -795,6 +795,7 @@ void CloneVoice::redo(EditData*)
 
 AddElement::AddElement(EngravingItem* e)
 {
+    DO_ASSERT_X(!e->generated(), String("Generated item %1 passed to AddElement").arg(String(e->typeName())));
```

Comment:
Ah, this should be 
```suggestion
    DO_ASSERT_X(!e->generated(), String("Generated item %1 passed to AddElement").arg(String::fromUtf8(e->typeName())));
```
That will make the "without Qt" check happy.

(What's happening now, is that it creates a QString from the const char* and then creates a String from the QString.)

Reply:
Oh, you just did... 
Somehow I keep commenting on GitHub pages that I apparently had to refresh 😅